src/lib/content.d.ts

```diff
@@ -163,6 +163,9 @@ interface GuVideoBlockElement {
 	html: string;
 	source: string;
 	role?: RoleType;
+	imageMedia?: { allImages: Image[] };
+	originalUrl?: string;
+	url?: string;
```

Comment:
I added these types after refreshing the fixture content and finding them missing